specification/app/resource-manager/Microsoft.App/preview/2025-10-02-preview/SessionPools.json

```diff
@@ -346,6 +346,89 @@
     }
   },
   "definitions": {
+    "TemplatePoolStatus": {
+      "description": "The status of pods in the pool of this template.",
+      "type": "object",
+      "properties": {
+        "expectedCount": {
+          "description": "The expected count of pods in this pool",
+          "type": "integer",
+          "format": "int32"
+        },
+        "readyCount": {
+          "description": "The ready count of pods in this pool",
+          "type": "integer",
+          "format": "int32"
+        },
+        "pendingCount": {
+          "description": "The pending count of pods in this pool",
+          "type": "integer",
+          "format": "int32"
+        },
+        "imagePullFailCount": {
+          "description": "The image pull fail count of pods in this pool",
+          "type": "integer",
+          "format": "int32"
+        },
+        "crashCount": {
+          "description": "The crash count of pods in this pool",
+          "type": "integer",
+          "format": "int32"
+        },
+        "allocatedCount": {
+          "description": "The allocated count of pods in this pool",
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "TemplateStatus": {
+      "description": "The status of the session pool template.",
+      "type": "object",
+      "properties": {
+        "details": {
+          "description": "The detailed status of this template.",
+          "type": "string"
+        },
+        "createdTime": {
+          "description": "The creation time of this template.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "status": {
+          "description": "The status of this template.",
+          "$ref": "#/definitions/TemplatePoolStatus"
+        },
+        "containers": {
+          "description": "List of container definitions for the sessions of this template.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SessionContainer"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "ingress": {
+          "description": "Session pool ingress configuration of this template.",
+          "$ref": "#/definitions/SessionIngress"
+        }
+      }
+    },
+    "TemplateUpdateStatus": {
+      "type": "object",
+      "properties": {
+        "activeTemplate": {
+          "description": "The status of the current active template.",
+          "$ref": "#/definitions/TemplateStatus"
+        },
+        "desiredTemplate": {
+          "description": "The status of the desired template during session pool update.",
+          "$ref": "#/definitions/TemplateStatus"
+        }
+      },
+      "additionalProperties": false
```

Comment:
why add this one?

Reply:
Thanks, removed, this should not be needed.